examples/cfd/darcy_transolver/train_transolver_darcy.py

```diff
@@ -0,0 +1,184 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 - 2024 NVIDIA CORPORATION & AFFILIATES.
```

Comment:
Is this taken from here? https://github.com/thuml/Transolver/blob/main/Physics_Attention.py
If so, the license header needs to change.

Reply:
Yes, I've already changed the license header.